api/environments/models.py

```diff
@@ -369,8 +381,41 @@ def _get_environment_document_from_db(
         cls,
         api_key: str,
     ) -> dict[str, typing.Any]:
-        environment = cls.objects.filter_for_document_builder(api_key=api_key).get()
-        return map_environment_to_environment_document(environment)
+        environment = cls.objects.filter_for_document_builder(
+            api_key=api_key,
+            extra_prefetch_related=[
+                Prefetch(
+                    "feature_states",
+                    queryset=FeatureState.objects.select_related(
+                        "feature",
+                        "feature_state_value",
+                        "identity",
+                        "identity__environment",
+                    ).prefetch_related(
+                        "identity__identity_traits",
```

Comment:
I'm unsure why we would need traits here?

Reply:
I thought I had fair reasoning for that, but, upon reviewing, I tend to agree with your question/comment. Removed both trait prefetching and trait mapping to the environment document.